.github/workflows/check.yaml

```
@@ -6,18 +6,18 @@ on:
 
 jobs:
   luacheck:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Please, let's use a concrete version of the ubuntu to avoid sudden breakdown of the CI the future.
```suggestion
    runs-on: ubuntu-24.04
```

---

fixed